### PR TITLE
Add missing test cases

### PR DESCRIPTION
**Pull request by PrinsFrank**

### CODE REVIEW

File: tests/Unit/Exception/UnableToCreateTransliteratorExceptionTest.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Exception;
```
Comment (szepeviktor) [inline tests/Unit/Exception/UnableToCreateTransliteratorExceptionTest.php]:
```suggestion
namespace PrinsFrank\Transliteration\Tests\Unit\Exception;
```

File: tests/Unit/FormalIdSyntax/Components/CharacterTest.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+namespace FormalIdSyntax\Components;
```
Comment (szepeviktor) [inline tests/Unit/FormalIdSyntax/Components/CharacterTest.php]:
```suggestion
namespace PrinsFrank\Transliteration\Tests\Unit\FormalIdSyntax\Components;
```

**Comment by szepeviktor** [comment]
How could it be that `composer dump-autoload --optimize --strict-psr` have not caught these?

https://github.com/PrinsFrank/transliteration/blob/d21622c3413977669f0d52109a9d6bca063a1b79/composer.json#L13

File: phpstan.neon [hunk 3]
```diff
@@ -14,6 +14,8 @@ parameters:
             - 'PHPUnit\Framework\Exception'
             - 'SebastianBergmann\RecursionContext\InvalidArgumentException'
     treatPhpDocTypesAsCertain: false
+    ignoreErrors:
+        - '#^Method PrinsFrank\\\\Transliteration\\\\Tests\\\\.* throws checked exception .* but it\'s missing from the PHPDoc @throws tag\\.$#'
```
Comment (szepeviktor) [inline phpstan.neon]: NEON does not have this feature
https://github.com/nette/neon/issues/43

Comment (szepeviktor) [inline phpstan.neon]:
```suggestion
        - "#^Method PrinsFrank\\\\Transliteration\\\\Tests\\\\.+ throws checked exception .+ but it's missing from the PHPDoc @throws tag\\.$#"
```

Comment (PrinsFrank) [reply phpstan.neon]: Thank you!!!

File: src/Rule/Components/VariableDefinition.php [hunk 4]
```diff
@@ -4,18 +4,39 @@
...
     public function __toString(): string
     {
-        return Literal::Dollar_Sign->value . $this->name . Literal::Space->value . Literal::Equals_Sign->value . Literal::Space->value . $this->value . Literal::Semicolon->value;
+        return Literal::Dollar_Sign->value
+               . $this->name
+               . Literal::Space->value
+               . Literal::Equals_Sign->value
+               . Literal::Space->value
+               . Character::escape($this->value)
+               . Literal::Semicolon->value;
```
Comment (szepeviktor) [inline src/Rule/Components/VariableDefinition.php]: No art in code please.
```suggestion
            . $this->name
            . Literal::Space->value
            . Literal::Equals_Sign->value
            . Literal::Space->value
            . Character::escape($this->value)
            . Literal::Semicolon->value;
```

Comment (PrinsFrank) [reply src/Rule/Components/VariableDefinition.php]: Haha, this is somehow how my IDE thinks indenting should work. ;)

**Pull request merged**